docs/E2ETESTING.md

```diff
@@ -0,0 +1,77 @@
+# End-to-End (E2E) Testing with Stark
```

Comment:
I suggest to re-order the topics like this:

**End-to-End (E2E) Testing with Stark**
- Protractor
--- Running end-to-end tests
--- Adding tests
- BrowserStack (optional)
--- Running end-to-end tests with BrowserStack
--- Common errors

What do you think?